config/settings.py

```diff
@@ -148,12 +148,10 @@
 
 try:
     database_url = os.environ["DATABASE_URL"]
-    DATABASES = {"default": dj_database_url.config(ssl_require=True)}
-    DATABASES["default"]["ENGINE"] = "django.contrib.gis.db.backends.postgis"
+    DATABASES = {"default": dj_database_url.config(ssl_require=not DEBUG)}
```

Comment:
Est-ce qu'on veut pas plutôt
```suggestion
    DATABASES = {"default": dj_database_url.config(ssl_require=(ENVIRONMENT != 'local'))}
```
?
Je peux imaginer faire tourner un test sur Scalingo avec DEBUG activé pour vérifier un truc, mais je veux quand même être en SSL dans ce cas.

Reply:
Si, bien sur